PIM/Troubleshooting/03_MissingStockValue.md

```diff
@@ -0,0 +1,58 @@
+[!!Warehouse](../../RetailSuiteWarehousing/Overview/01_General.md)
+[!!User Interface Product list](../UserInterface/02a_List.md)   
+[!!User Interface Warehouse](../../RetailSuiteWarehousing/UserInterface/00_UserInterface.md)
+[!!Data type list](../UserInterface/04_DataTypeList.md)
+
+
+# Add a missing stock value
+
+#### Error Description
+
+In a products attribute field with the *Stock Value* data type is no stock displayed, although a stock is specified in the *Warehouse* module.
+
+Follow the instructions below to display the stock in the product view.
+
+#### Prerequisites
+
+No prerequisites to fulfill.
+
+#### Procedure
+*PIM > Products > Tab LIST*
+
+![Products](../../Assets/Screenshots/PIM/Products/List/Products.png "[Products]")
+
+1. Click the product to which the stock is not displayed in the product list.
+  The *Edit Product* view is displayed.
+
+  ![Attributes](../../Assets/Screenshots/PIM/Products/List/Attributes/AttributesEdit.png "[Attributes]")
+
+2. Switch to the *UCS Warehouse* tab.   
+
+  ![UCS Warehouse](../../Assets/Screenshots/PIM/Products/List/UCSWarehouse/WarehousingNotActive.png "[UCS Warehouse]")
+
+3. Check if the warehousing is active. If the warehousing is inactive, click the [ACTIVATE WAREHOUSING] button to activate the warehousing for the selected product.   
```

Comment:
```suggestion
3. Check if the warehousing is active. If the warehousing is not active, click the [ACTIVATE WAREHOUSING] button to activate the warehousing for the selected product.   
```
Im Bildschirm steht Warehousing not active, daher würde ich auch diese Wörter verwenden. Warum "warehousing" und nicht "warehouse"?